account_asset_management/models/account_asset.py

```diff
@@ -793,15 +800,25 @@ def _compute_line_dates(self, table, start_date, stop_date):
         # last entry
         if not (self.method_time == 'number' and
                 len(line_dates) == self.method_number):
-            line_dates.append(line_date)
+            if self.days_calc:
+                line_dates.append(stop_date)
+            else:
+                line_dates.append(line_date)
 
         return line_dates
 
-    def _compute_depreciation_amount_per_fiscal_year(self, table, line_dates):
+    def _compute_depreciation_amount_per_fiscal_year(
+            self, table, line_dates, depreciation_start_date,
+            depreciation_stop_date):
         digits = self.env['decimal.precision'].precision_get('Account')
         fy_residual_amount = self.depreciation_base
         i_max = len(table) - 1
         asset_sign = self.depreciation_base >= 0 and 1 or -1
+        day_amount = 0.0
+        if self.days_calc:
+            days = (depreciation_stop_date - depreciation_start_date).days + 1
+            day_amount = round(self.depreciation_base / days, digits)
```

Comment:
This rounding should not be applied. I have checked with this page and the numbers of daily depreciation do match between Microsoft and Odoo when this rounding is removed.

In Odoo, after removing the rounding.
![image](https://user-images.githubusercontent.com/7683926/67053572-1a20bf00-f142-11e9-8c1c-e2ff195c81f3.png)

In MS:
![image](https://user-images.githubusercontent.com/7683926/67053631-3d4b6e80-f142-11e9-9545-3f078d527070.png)

See it in my backport to 11.0 https://github.com/OCA/account-financial-tools/pull/902/commits/16de4642480e9c17a361f4d21765830e0179c0b3